### PR TITLE
Page routing configuration

### PR DESCRIPTION
**Pull request by evie-adenoyin**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -19,6 +25,11 @@ function App() {
           </Route>
           <Route path="/support" element={<Support />} />
           <Route path="/energy-analytic" element={<EnergyAnalytic />} />
+          <Route path="/" element={<SignUp />} />
+          <Route path="signin" element={<SignIn />} />
+          <Route path="verification/code" element={<Verification />} />
+          <Route path="details" element={<Detail />} />
+          <Route path="forgot/password/" element={<ForgotPasswordPage />} />
```
Comment (slimsolz) [inline src/App.js:32]:
```suggestion
          <Route path="forgot/password/" element={<ForgotPasswordPage />} /> 
          <Route path="forgot-password/" element={<ForgotPasswordPage />} />
```

File: src/App.js [hunk 2]
```diff
@@ -19,6 +25,11 @@ function App() {
           </Route>
           <Route path="/support" element={<Support />} />
           <Route path="/energy-analytic" element={<EnergyAnalytic />} />
+          <Route path="/" element={<SignUp />} />
+          <Route path="signin" element={<SignIn />} />
+          <Route path="verification/code" element={<Verification />} />
```
Comment (slimsolz) [inline src/App.js:30]: We only have one verification page so I think removing the `/code` isn't a bad idea. what do you think?
```suggestion
          <Route path="verification/code" element={<Verification />} />
          <Route path="verification" element={<Verification />} />
```

**Comment by slimsolz** [comment]
Please resolve the conflict @evie-adenoyin

**Comment by slimsolz** [comment]
please review the tiny updates i made, if all is good then we can merge this branch @evie-adenoyin

**Comment by slimsolz** [comment]
LGTM

**Review by evie-adenoyin — COMMENTED** [review]
LGTM...  i was just all about the conflict... narrow me

**Pull request merged**